star.json not found

Here:
```js
Bundling Meteor app...
Generating the index.html...
module.js:338
    throw err;
    ^

Error: Cannot find module '/Users/lukeadams/Projects/belljar-io/Users/lukeadams/Desktop/buildtest/bundle/star.json'
```
It has to do with relative URL handling
PR incoming...

--- a/meteor.js
+++ b/meteor.js
@@ -7,7 +7,7 @@ var spinner = require('simple-spinner');
 // VARIABLES
 var argPath = process.argv[2],
     basePath = './',
-    buildPath = basePath + argPath,
+    buildPath = path.resolve(argPath),
     bundleName = path.basename(path.resolve(basePath));
 
 // execute shell scripts

